2022/incoming/sleigh-odometer.pod

```diff
@@ -0,0 +1,105 @@
+Author: Thibault DUPONCHELLE <thibault.duponchelle@gmail.com>
+Title: The Sleigh odometer
+Topic: state
+
+=encoding utf8
+
+=head1 Sleigh odometer
+
+=head2 Customize the sleigh
+
+It's already December and elves are working hard to produce all toys for Christmas.
+
+On the other side, Santa is preparing his Sleigh, cleaning or customizing.
+ 
+This year, in addition to a temperature sensor, Santa asked to install an odometer to his sleigh so that he can count each delivery.
+For the record, the distance unit is not kilometers or miles nor inches but "elf little toe"... but it does not change what follows :)
```

Comment:
Not sure, but maybe "not kilometers nor miles nor inches" would read better?

Reply:
Fixed